spec/support/service_spec_helper.rb

```diff
@@ -1,4 +1,16 @@
 module ServiceSpecHelper
+  RSpec.configure do |config|
+    config.around(:example, :type => :service) do |example|
+      default_tenant = Tenant.first_or_create!(:external_tenant => default_account_number)
+
+      ActsAsTenant.with_tenant(default_tenant) do
```

Comment:
@eclarizio For services would we be forced to using the default_tenant all the time. Could the service specs use other tenants?

Reply:
If they needed to use a different tenant then they would have to just wrap their example in another `ActsAsTenant.with_tenant` block, I think.

Example:
```
let(:new_tenant) { create(:tenant, :external_tenant => 123) }

it "works with a different tenant" do
  ActsAsTenant.with_tenant(new_tenant) do
    # ...expectations here
  end
end
```